src/app/containers/MediaPlayer/index.jsx

```diff
@@ -76,7 +76,7 @@ const MediaPlayerContainer = ({
     return null; // this should be the holding image with an error overlay
   }
 
-  const placeholderSrc = getPlaceholderSrc(imageUrl);
+  const placeholderSrc = getIChefURL({ originCode, locator, resolution });
```

Comment:
In this new implementation, I believe when the `originCode` is not `mpv` instead of getting a placeholder value, we get an iChefURL. Is this desired?

Reply:
I moved the logic for `getPlaceholderSrc` to `getIchefUrl` since the placeholder src is actually an ichef url with some resolution computation done on it. The value of `placeholderSrc` remains the same.